src/DappLink.jsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
```

Comment:
Not sure this licence header aligns with what we have in the rest of the repo, nor is GPL 3 correct for this one?